L/libblastrampoline/build_tarballs.jl

```diff
@@ -3,16 +3,19 @@
 using BinaryBuilder
 
 name = "libblastrampoline"
-version = v"3.1.0"
+version = v"4.0.0"
 
 # Collection of sources required to build libblastrampoline
 sources = [
-    GitSource("https://github.com/staticfloat/libblastrampoline",
-              "c6c7bc5d4ae088bd7c519d58e3fb8b686d00db0c")
+    GitSource("https://github.com/JuliaLinearAlgebra/libblastrampoline",
```

Comment:
I'm away from computer, but I'd try 
```suggestion
    GitSource("https://github.com/JuliaLinearAlgebra/libblastrampoline.git",
```
We have some dodgy assumptions about the fact the URL should end with `.git`

Reply:
Uhm, no, reading the full log I'm not sure that's the issue. Maybe try cleaning up the cached repository on the build machine?

Reply:
It was indeed the missing `.git` in the URL.  We should fix those assumptions, in the meantime, adding the trailing `.git` is a working fix